contrib/triton-aml/Dockerfile

```diff
@@ -0,0 +1,96 @@
+# It is recommended to use a machine which supports CUDA to build this image.
+FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04 AS BUILDER
+RUN apt-get update --fix-missing
+RUN apt-get install -y curl git autoconf automake libtool curl make g++ unzip cmake build-essential cpio
+RUN apt-get -y clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# install zlib
+WORKDIR /
+RUN git clone --no-checkout https://github.com/madler/zlib
+WORKDIR /zlib
+RUN git checkout tags/v1.2.10 && \
```

Comment:
We have zlib in `src/3rd_party/zlib`. It needs to be installed for getting a specific version or because protobuf requires it?

Reply:
> We have zlib in `src/3rd_party/zlib`. It needs to be installed for getting a specific version or because protobuf requires it?

I think it's for getting a specific version that works well for the Office team. We got this from their original Dockerfile.